src/Utils/Hooks/useMap.ts

```diff
@@ -38,6 +39,10 @@ export const useMap=(mapContainer:React.RefObject<HTMLDivElement>,apikey:string,
                 }
             )
 
+            const ui = H.ui.UI.createDefault(mapInstance.current, defaultLayers)
+            ui?.removeControl('mapsettings')
```

Comment:
The Zoom should be controlled by the Radius not manually. Check Here Code for Delta ;)

Reply:
what is Delta?